### PR TITLE
Update project docs:

### PR DESCRIPTION
**Pull request by dmitry-timofeev**
## Overview

- Update the EJB description to reflect multiple services support
- Don't mention alpha — roadmap is more appropriate for that
- Remove info on required Java version as its not much relevant
  there (and is provided in LC README and CG)
- Update link to OpenJDK to 12 as 10 is EOL

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Changelog is updated if needed (in case of notable or breaking changes)
- [x] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 Exonum Java Binding is a framework for building blockchain applications in Java, 
 powered by Exonum.
 
-The project is at alpha stage. Currently you can define a stateful Exonum Service 
-in Java, configure a network of nodes and run the service in it.
+With Exonum Java Binding you can define a stateful Exonum Service in Java,
```
Comment (bullet-tooth) [inline exonum-java-binding/README.md]:
```suggestion
With Exonum Java Binding you can define stateful Exonum Services in Java,
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ is valid. It may take a long time, so you can pass `--skip-tests` flag to skip t
 $ ./package_app.sh --skip-tests
 ```
 
-By default, the EJB App is build in debug mode, which affects performance and not desired for production usage. 
+By default, the EJB App is built in debug mode, which affects performance and not desired for production usage.
```
Comment (Maria-Nosyk) [inline CONTRIBUTING.md]: and is not desired

File: exonum-java-binding/README.md (hunk 1 above)
Comment (Maria-Nosyk) [reply exonum-java-binding/README.md]: agree with the suggestion

**Comment by dmitry-timofeev** [comment]
> Why don't we change travis job that runs on openjdk11 to openjdk12?

I think it is better to be discussed separately if we shall test only on 8 and the most recent, or on 8 and the latest LTS (the LTS status, however, is designated by each vendor, the OpenJDK 11 builds Travis uses are no longer supported, Oracle/AdoptOpenJDK/<some other vendors who adopted Oracle's LTS status> are).

**Pull request merged**